test.js

```diff
@@ -342,7 +343,11 @@ test('flush - respond with an error', async t => {
     }
   ]
 
-  await t.throws(client.flush(), 'Bad Request')
+  try {
+    await t.throwsAsync(() => client.flush())
+  } catch (error) {
+    console.log('Dafuq:', error)
```

Comment:
Part of debugging in an attempt to find where the unhandled promise rejection was coming from.
This console log is not called at all. Very confused